sktime/forecasting/compose/_ensemble.py

```diff
@@ -114,6 +115,9 @@ def __init__(
             forecasters=forecasters,
             n_jobs=n_jobs,
         )
+        if method and regressor:
+            raise ValueError("method and regressor can not be used together!")
+        self.method = method
```

Comment:
I think we need a default for `method` which is not None and also a `method="feature-importances"`.

Reply:
Do you suggest:
1. if `method` and `regressor` both are not given, we go to `method="feature-importance"` and Boosting regressor.
2. If `method` is given as `feature-importance`, we go to  `method="feature-importance"` and Boosting regressor.
3. If `method` is given as `feature-importance` and regressor is given, we go to `method="feature-importance"` and given regressor.
4. If `method` is given as others, we go to `method`
5. If `method` is given as others and regressor is also given, raise an error.
Maybe a little confusing?

Reply:
I would say lets default to `method="inverse-variance"` and not None as it will be faster to compute also. For each of the methods, we have to check that all needed params are given or setted (in case of None e.g.).

Reply:
I think then things are more clear for user and for us to implement?